client/src/elm/Translate.elm

```diff
@@ -13085,6 +13096,26 @@ translationSet trans =
             , kinyarwanda = Nothing
             }
 
+        SendViaWhatsAppExecutionResultFailure ->
+            { english = "Action Failed. Please try again. If problem persisits, please contact system administrator."
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppExecutionResultPleaseWait ->
+            { english = "Please allow up to one minute for action to complete ..."
```

Comment:
```suggestion
            { english = "Please allow up to one minute for action to complete..."
```

Reply:
This phrase was removed in follow up PR, so not point i changing this.